yt/__init__.py

```diff
@@ -17,77 +17,72 @@
 __version__ = "4.0.dev0"
 
 # First module imports
-import numpy as np  # For modern purposes
 import numpy  # In case anyone wishes to use it by name
+import numpy as np  # In case anyone wishes to use it by name
```

Comment:
note to reviewers: the comment duplication here is an unfortunate side effect (I dare say bug) in isort, likely because it gets confused that we import the same module twice...

Though this has nothing to do with the original scope of this PR, I would like to propose we remove these imports completely: unless I'm missing the point completely, I do not think it is wise to encourage users to import numpy FROM yt, which is the only thing these lines get u, seems to me.

Reply:
bug confirmed upstream (note that I reported it with a different version of isort, so it's not exactly a reproduction of what we're seeing here): https://github.com/PyCQA/isort/issues/1631#issuecomment-753828006

Reply:
I think it's OK to not import it here; I briefly thought maybe it might have implications for either ensuring numpy is imported (it does set up a global state) or making sure that it's accessible to the plugins, but I think in both cases it's not actually relevant.